CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+- Add API documentation for `realtimeSubscribeToAttendeeIdPresence` callback parameters.
```

Comment:
nit: Should we mention that you add the type `RealtimeSubscribeToAttendeeIdPresenceCallback`?

Reply:
Thanks for the catch! Thats right. Updated it.